spec/support/shared_examples/basic_destroy_endpoint.rb

```diff
@@ -3,7 +3,7 @@
     expect { req }.to change(model_klass, :count).by(-1)
   end
 
-  it 'returns status code ok', dictum: DICTUM_DESTROY_DESCRIPTION do
+  it 'returns status code ok' do
```

Comment:
This won't work as a fictium documentation example, as it uses req inside the block.

I suggest, the following for this:

```rb
shared_examples 'basic destroy endpoint' do
   describe example 'with valid information' do
       before do
         req
       end

       default_example
   end
```

This will automatically add an example for that case, and it will be properly documented.

Reply:
I actually forgot to adapt this to Fictium :eyes: